src/routes/dashboard/(admin)/labs/QuotaForm.svelte

```diff
@@ -47,22 +62,47 @@
         <tr>
           <th>Laboratory</th>
           <th class="table-cell-fit">Quota ({total})</th>
+          <th class="table-cell-fit">Delete/Restore</th>
         </tr>
       </thead>
       <tbody>
-        {#each labs as { id, name, quota } (id)}
+        {#each labs as { id, name, quota, deletedAt } (id)}
           {@const placeholder = quota.toString()}
           <tr>
-            <td class="!align-middle">{name}</td>
+            <td class="!align-middle">
+              {#if deletedAt === null}
+                {name}
+              {:else}
+                <strike>
+                  {name}
+                </strike>
+              {/if}
+            </td>
             <td class="table-cell-fit"
               ><input
                 type="number"
                 min="0"
                 name={id}
                 {placeholder}
+                disabled={deletedAt !== null}
                 class="input variant-form-material px-2 py-1"
               /></td
             >
+            <td class="table-cell-fit">
+              {#if deletedAt === null}
+                <button
+                  formaction="?/delete"
```

Comment:
Let's prefer the fully formed path here.
```suggestion
                  formaction="/dashboard/labs/?/delete"
```